mast_aladin/app_sidecar.py

```diff
@@ -127,19 +149,55 @@ def open(
                 "run `pip install jdaviz`.",
                 UserWarning
             )
+        return apps
 
-        n_columns = len(apps)
+    def _attach_sidecars(self, apps, anchor, title):
+        """
+        Attach apps to sidecars. If only one anchor, all apps share
+        a single sidecar. Otherwise, create one sidecar per app.
+        In the multiple case, each sidecar `n` will reference the
+        `n-1` sidecar instance.
+        """
+        if not isinstance(anchor, list):
+            anchor = [anchor]
+
+        if len(anchor) == 1:
+            # shared single sidecar
+            ctx = UpstreamSidecar(anchor=anchor[0], title=title)
+            for app in apps:
+                app.sidecar = ctx
+
+        else:
+            # multiple sidecars
+            anchor = self._normalize_anchor(anchor, apps)
+            ref = None
+            for app, anc in zip(apps, anchor):
+                ctx = UpstreamSidecar(anchor=anc, title=title, ref=ref)
+                app.sidecar = ctx
+                ref = ctx
```

Comment:
This pattern fails on the first Sidecar init using sidecar v0.8.0 because of a bug that I introduced upstream. Bugfix PR: https://github.com/jupyter-widgets/jupyterlab-sidecar/pull/146